util/browser.ts

```diff
@@ -4,8 +4,8 @@ export const isMac =
     : false
 
 export const getAbsoluteURL = (path = '') => {
-  const baseURL = process.env.VERCEL_URL
-    ? `https://${process.env.VERCEL_URL}`
+  const baseURL = process.env.NEXT_PUBLIC_VERCEL_URL
```

Comment:
Just for my own curiosity: we are doing the same thing, why is your way better?

Reply:
Few reasons:

1. We dont need .env.development just yet as we only have one env-specific configuration, so we can just default to it here.
2. If we ever change how the URL is constructed or the variable name, we will only need to modify getAbsoluteURL util, instead of going to every place it was used.
3. The .env.production is also redundant as that's the environment variable automatically exposed by Vercel.

So, this actually fixes the original bug that was there, instead of working around it with dotfiles.

Reply:
No there was two bugs: the wrong variable used (which is also fixed here), and the wrong setup setup in vercel (which has been fixed in our settings).
The production env is not necessary indeed, but the development one is just another way to do what you are doing in your function to make it work locally.
But fine with me, just was wondering if there was something intrinsically better.